src/modules/NtpClient.rb

```diff
@@ -1018,6 +984,9 @@ def read_ad_address!
       end
     end
 
+    # Reads /etc/sysconfig/ntp NTPD_RUN_CHROOTED and sets @run_chroot as true
```

Comment:
This comment is a matter of personal taste, so take it with a grain of salt. When documenting, I prefer to _summarize_ in the first line what the method does, clarifying below if needed. In this case, you could also write something like:

```
# Set @run_chroot according to NTPD_RUN_CHROOTED value found in /etc/sysconfig/ntp
#
# @return [Boolean] true when value is "yes"; false in any other case.
```